python/housinginsights/ingestion/LoadData.py

```diff
@@ -312,6 +313,7 @@ def _create_zone_facts_table(self):
         # create empty zone_facts table
         metadata = MetaData(bind=self.engine)
         zone_facts = Table('zone_facts', metadata,
+                           Column('zone_type', String),
```

Comment:
So, at a minimum: add a code comment here that says 'also update meta.json when adding/removing columns', and then add the appropriate meta.json fields. Or more cleanly, refactor this create table statement to use meta.json.